pynicotine/gtkgui/ui/settings/search.ui

```diff
@@ -185,24 +227,26 @@
                 <property name="visible">1</property>
                 <property name="focusable">0</property>
                 <child>
-                  <object class="GtkLabel">
-                    <property name="visible">1</property>
-                    <property name="label" translatable="yes">Include:</property>
-                    <property name="xalign">0</property>
-                    <property name="wrap">1</property>
-                    <property name="mnemonic_widget">FilterIn</property>
-                  </object>
-                </child>
-              </object>
-            </child>
-            <child>
-              <object class="GtkFlowBoxChild">
-                <property name="visible">1</property>
-                <property name="focusable">0</property>
-                <child>
-                  <object class="GtkEntry" id="FilterIn">
+                  <object class="GtkBox">
                     <property name="visible">1</property>
-                    <property name="tooltip-text" translatable="yes">Filter in results whose file paths contain the specified text. Multiple phrases and words can be specified, e.g. exact phrase|music|term|exact phrase two</property>
+                    <property name="focusable">0</property>
+                    <child>
+                      <object class="GtkLabel">
+                        <property name="visible">1</property>
+                        <property name="label">            </property>
```

Comment:
We shouldn't do this. It's not reliable with different themes and fonts.

Reply:
I can't get the spacing property to work. Indeed I tried on several themes and it is slightly different on each.

Reply:
The spacing property is for spacing between elements, margin-start/end adds spacing to the left and right of an element.

In any case, we should probably give this some thought before adding it. It would look out of place to only include such spacing in the search category.

Reply:
Thank you, perhaps in future I can look into doing the widget indentation of the settings categories overall, I agree it is best to avoid these awkward individual style hacks, so I have removed the label spacers.